include/openenclave/internal/crypto/keys.h

```diff
@@ -7,7 +7,7 @@
 #include <openenclave/bits/defs.h>
 #include <openenclave/bits/result.h>
 #include <openenclave/bits/types.h>
-#include "sgxtypes.h"
```

Comment:
keys.h is an SGX internal helper method for `EGETKEY`, not a crypto API implemented by the crypto lib.

Extra credit: we should probably rename keys.h to reflect that e.g. `sgxkeys.h` like the existing `sgxsign.h` or `sgxproperties.h`

Reply:
Done. Including extra credit.